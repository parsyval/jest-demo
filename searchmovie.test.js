const searchmovie = require('./searchmovie.js');

it('Returns the movies that matches the query', () => {
	expect(searchmovie('F')).toMatchSnapshot();
});
