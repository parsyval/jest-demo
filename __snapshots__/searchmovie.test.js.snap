// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Returns the movies that matches the query 1`] = `
Array [
  Object {
    "tags": "kids,animation",
    "title": "Frozen",
  },
  Object {
    "tags": "kids,animation,fish",
    "title": "FindingNemo",
  },
  Object {
    "tags": "kids,animation,suite",
    "title": "Frozen2",
  },
]
`;
